Modules/MUON/MID/src/DigitsQcCheck.cxx

```diff
@@ -120,14 +120,20 @@ Quality DigitsQcCheck::check(std::map<std::string, std::shared_ptr<MonitorObject
   for (auto& item : *moMap) {
     if (item.second->getName() == "LocalBoardsMap") {
       if (mHistoHelper.getNTFs() > 0) {
-        int nEmptyLB = 0;
-        int nBadLB = 0;
+        nEmptyLB = 0;
+        nBadLB = 0;
+        maxVal = 0;
+        minVal = 1000;
         auto histo = static_cast<TH2F*>(item.second->getObject());
         mHistoHelper.normalizeHistoTokHz(histo);
         for (int bx = 1; bx < 15; bx++) {
           for (int by = 1; by < 37; by++) {
             if (!((bx > 6) && (bx < 9) && (by > 15) && (by < 22))) { // central zone empty
               double val = histo->GetBinContent(bx, by);
+              if (val > maxVal)
```

Comment:
In general it is recommended to always use `{` and `}` even when there is a single line.